webpack.config.js

```diff
@@ -99,6 +99,7 @@ const common = {
         },
         modulesDirectories: ['node_modules', 'vendor_modules'],
         root: [p.kendo],
+        extensions: ['', '.js', '.jsx'], // allows importing these file types without extensions
```

Comment:
this looks like it doesn't include the defaults....

https://webpack.github.io/docs/configuration.html#resolve-extensions